docs/content/en/docs/getting-started/production-environment/baremetal-getstarted.md

```diff
@@ -0,0 +1,134 @@
+---
+title: "Create Bare Metal production cluster"
+weight: 40
+---
+
+EKS Anywhere supports a Bare Metal provider for production grade EKS Anywhere deployments.
+EKS Anywhere allows you to provision and manage Amazon EKS on your own infrastructure.
```

Comment:
> manage Amazon EKS on your own infrastructure

To me this sounds like, you would able to manage EKS on the cloud from your own infra using EKS-A.

I would re-phrase it to maybe say something along these lines
```suggestion
EKS Anywhere allows you to provision and manage Amazon EKS like Kubernetes environment in your own infrastructure.
```

Reply:
How about this: "EKS Anywhere allows you to provision and manage Kubernetes clusters based on Amazon EKS software on your own infrastructure."
Feel free to make other wording suggestions.